src/crewai/task.py

```diff
@@ -53,9 +53,9 @@ class Task(BaseModel):
     i18n: I18N = I18N()
     name: Optional[str] = Field(default=None)
     prompt_context: Optional[str] = None
-    description: str = Field(description="Description of the actual task.")
-    expected_output: str = Field(
-        description="Clear definition of expected output for the task."
+    description: str = Field(description="Description of the actual task."
+    )
```

Comment:
Should these `)` be up?

Reply:
I moved up description but my formatter keeps moving the rest down.